jobs/ci-run/build/streams.yaml

```diff
@@ -179,7 +108,7 @@
           # NOTE: this sync removes the "build-$SHA" indirection and effectively
           # pulls the binaries one folder up (under agent/)
           ${SCRIPTS_DIR}/azure_publish_tools.py sync \
-            ${WORKSPACE}/agent/build-${SHORT_GIT_COMMIT}/ \
+            ${WORKSPACE}/agent/build-${JUJU_VERSION}-${SHORT_GIT_COMMIT}/ \
```

Comment:
Why do we need to add juju version? The git sha is all that's needed right?

Reply:
It is needed as it has the build number in it and re-running jobs with different build numbers will not always use the correct artifacts. Doing it this way ensures that the correct artifacts are used through out the whole run no matter what.